sonar-fsharp-plugin/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
```

Comment:
could u had a section with developers and include yourself there

  <developers>
    <developer>
      <id></id>
      <name></name>
      <email></email>
    </developer>

Reply:
Thanks, the section is in the parent `pom.xml` This is inheritted, in the SonarQube marketplace the organization jmecsoftware is already displayed.